### ISSUE TITLE
Videos UI: Styling and layout across all media widths needs improvement.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#57799]
Our styling and layout across all media widths needs improvement. Mobile looks generally pretty good, but we move to two columns too soon. Let's get our layouts and proportions making sense first for this issue (so that it can be launched), and we'll more high-fidelity "pixel perfect" details after launch in https://github.com/Automattic/wp-calypso/issues/57793.

| 376px | 765px | 1021px | 1851px | 2559px |
| --- | --- | --- | --- | --- |
| <img width="376" alt="Screen Shot 2021-11-08 at 2 18 37 PM" src="https://user-images.githubusercontent.com/349751/140828511-d1c8eba9-1381-44ff-9696-693c1661e4ae.png"> | <img width="765" alt="Screen Shot 2021-11-08 at 2 20 44 PM" src="https://user-images.githubusercontent.com/349751/140828519-e55e0ec3-95ee-40b9-9707-974791dac465.png"> | <img width="1021" alt="Screen Shot 2021-11-08 at 2 21 57 PM" src="https://user-images.githubusercontent.com/349751/140828739-745c2ac2-2671-4f75-8df8-87eaeddfe8bc.png"> | <img width="1536" alt="Screen Shot 2021-11-08 at 2 25 44 PM" src="https://user-images.githubusercontent.com/349751/140828745-f49f278c-0a7c-4e81-aaec-6900e2ffccce.png"> | <img width="2559" alt="Screen Shot 2021-11-08 at 2 27 28 PM" src="https://user-images.githubusercontent.com/349751/140828757-3d213527-3e4d-42f5-a8b8-c644286e1894.png"> |

**Comment by kwight** [comment]
@ollierozdarz  Do you have a preference for a breakpoint at which we go from single to two columns? Maybe 960px? Calypso breakpoints are here: https://github.com/Automattic/wp-calypso/blob/38c37bbfe35b8c741c4e346637987ddc0fb3ab59/client/assets/stylesheets/shared/mixins/_breakpoints.scss#L10 

<img width="1055" alt="Screen Shot 2021-11-08 at 2 43 19 PM" src="https://user-images.githubusercontent.com/349751/140829883-ef79da5b-9867-4e93-bb4c-a5a5aa80b86a.png">

I'm also wondering how you want us to handle large desktops;  should we give the UI a max-width, and then just center it regardless of screen width above that?

**Comment by ollierozdarz** [comment]
@kwight for the breakpoint, I think `1040px` will be nice. Anything smaller starts to get a bit funny. 

The design is intended to sit in a centered max-width container of approximately 1160px. If we need to go out to 1280px (from the break-points above), that's Ok too. On XL desktops, this won't give us any issues 🙂

**Comment by blackjackkent** [comment]
Actually it looks like the standard set of breakpoints are now coming from Gutenberg ( https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#media-queries ) which is slightly different numerically - 

```
$break-huge: 1440px;
$break-wide: 1280px;
$break-xlarge: 1080px;
$break-large: 960px;	// admin sidebar auto folds
$break-medium: 782px;	// adminbar goes big
$break-small: 600px;
$break-mobile: 480px;
$break-zoomed-in: 280px;
```

@ollierozdarz are you OK with 1080 instead for that break?

**Comment by blackjackkent** [comment]
In working on this, I also found that while the video content looks fine in a single column up to 1080px, the header content ("Watch five videos..." etc.) ended up with too much empty space at any size above roughly `$break-medium` (782px). In the PR I just created, I set different breakpoints for the header and video content accordingly. 

I've added screenshots for all these cases to the PR at https://github.com/Automattic/wp-calypso/pull/57857 -- @ollierozdarz and @kwight  can you give your opinion when you have a sec?

**Comment by ollierozdarz** [comment]
> @ollierozdarz are you OK with 1080 instead for that break?

@kwight yep, sounds great.

**Comment by kwight** [comment]
> In the PR I just created, I set different breakpoints for the header and video content accordingly.

Yep, that all makes sense, good call. Let's get #57857 merged, and we can keep iterating! 🎉

**Comment by kwight** [comment]
Closed in #57857 – layout and styling now at least make sense and look good at all widths. We'll use #57868 and #57867 to get everything even more inline with the original Figma mockups. cc/ @ianstewart @ollierozdarz
